src/Chapter1.hs

```diff
@@ -619,8 +637,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let tensDigit = div n 10
+        onesDigit = mod n 10
```

Comment:
One hint to make your solution a bit shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(first, second) = divMod n 10
```

You can see how we could pattern match on the pair 🙂 

And also here you also would need to consider case with negative numbers 🙂